content/en/blog/_posts/2023-12-14-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -39,7 +39,7 @@ and what changes you will need to make.
 These updated default settings affect a large proportion of Kubernetes users,
 and **will require changes** for users who were previously using the in-tree
 provider integrations. The legacy integrations offered compatibility with
-Azure, AWS, GCE, OpenStack, and vSphere; however for AWS and OpenStack the
+Azure, AWS, GCE, OpenStack, and vSphere; however for OpenStack and AWS the
 compiled-in integrations were removed in Kubernetes versions 1.26 and 1.27,
```

Comment:
We use alphabetical order for 3rd parties, so:
```suggestion
Azure, AWS, GCE, OpenStack, and vSphere; however for AWS and OpenStack the
compiled-in integrations were removed in Kubernetes versions 1.27 and 1.26,
```

Reply:
Any reason why we also change the versions order to be descendant?

Reply:
@ricardoamaro listing AWS before OpenStack, the removals happened in versions 1.27 and 1.26 respectively. If we'd removed OpenStack in 1.18, the text would be:
```markdown
Azure, AWS, GCE, OpenStack, and vSphere; however for AWS and OpenStack the
compiled-in integrations were removed in Kubernetes versions 1.27 and 1.18,
```
but we didn't.